src/UiTPAS/Event/EventProcessManager.php

```diff
@@ -76,6 +76,7 @@ private function handleEventCardSystemsUpdated(EventCardSystemsUpdated $eventCar
 
         $uitPasLabels = $this->uitpasLabelsRepository->loadAll();
 
+        // Create a list of UiTPAS labels that are supposed to be on the event.
```

Comment:
I wonder if these comments can be replaced by a descriptive private method 🤔

Reply:
Good idea, I'll pick it up tomorrow / Wednesday but if you have some time beforehand feel free to do it yourself if you like.